src/xcb_util.cpp

```diff
@@ -234,3 +234,42 @@ ewmh_change_desktop (xcb_connection_t *c, xcb_window_t root, uint destkop_id)
   // of this spec, in which case the timestamp field should be ignored.
   send_xcb_message (c, root, "_NET_CURRENT_DESKTOP", { destkop_id });
 }
+
+xcb_keycode_t *
+get_keycodes (xcb_connection_t *c, xcb_keysym_t sym) // const char * name)
+{
+  /**
+   * 1. Find a keysym for keyname in struct
+   * 2. Find possible keycodes for keysym
+   * 3. Return a vector of results.
+   */
```

Comment:
You should place docs above function definition

Reply:
Will correct

Reply:
TODO: error handling
P.S. is it considered OK writing things like that in conversations?